src/lang/Messages.properties

```diff
@@ -582,6 +582,16 @@ callback.options.label.rndport 			= Random Port:
 callback.options.label.port 			= Specify Port:
 callback.options.label.testurl 			= Test URL:
 callback.test.msg						= Received test callback: {0} from {1}
+callback.panel.clear.button.label		= Clear
+callback.panel.clear.button.toolTip		= Clear Callbacks
+callback.panel.options.button.label		= Options
+callback.panel.table.column.date		= Date
+callback.panel.table.column.method		= Method
+callback.panel.table.column.url			= URL
+callback.panel.table.column.handler		= Handler
+callback.panel.table.column.referer		= Referer
+callback.handler.none.name				= No callback handler
+callback.handler.test.name				= Testhandler
```

Comment:
`Test handler`?

Reply:
Good catch, I should have seen that yesterday. If it's changed to two words it should probably be [Title Caps](https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#style-guidelines).